hw/ip/otbn/doc/_index.md

```diff
@@ -246,42 +246,45 @@ GPRs are accessible from the base instruction subset, and WDRs are accessible fr
 
 ### Wide Special Purpose Registers (WSRs)
 
-In addition to the Wide Data Registers, BN instructions can also access WLEN-sized special purpose registers, short WSRs.
+OTBN has 256b Wide Special purpose Registers (WSRs).
+These are analogous to the 32b CSRs, but are used by the wide side of the processor.
+They can be accessed with the [`BN.WSRRS`]({{< relref "hw/ip/otbn/doc/isa#bnwsrrs" >}}) and [`BN.WSRRW`]({{< relref "hw/ip/otbn/doc/isa#bnwsrrw" >}}) instructions.
 
 <table>
   <thead>
     <tr>
       <th>Number</th>
-      <th>Privilege</th>
+      <th>Name</th>
+      <th>R/W</th>
       <th>Description</th>
     </tr>
   </thead>
   <tbody>
     <tr>
       <td>0x0</td>
+      <td><strong>MOD</strong></td>
       <td>RW</td>
-      <td>
-        <strong>MOD</strong>
-        Modulus.
-        To be used in the BN.ADDM and BN.SUBM instructions.
-        This WSR is mapped to the MOD0 to MOD7 CSRs.
-      </td>
+<td>
```

Comment:
Nit: Fix indentation of the td elements? You need to use HTML links if you'd like to avoid the hoops you're currently jumping through to get Markdown within HTML rendering :-)

Reply:
That was intentional, because Emacs's syntax highlighting was getting confused. I think it viewed the indented `</td>` element as the start of a code section. I can re-indent if you think it would be better, but I wondered whether other editors would have similar problems.